bentoml/saved_bundle/bundler.py

```diff
@@ -49,7 +53,9 @@ def save_to_dir(bento_service, path, version=None, silent=False):
     is not empty, this call may override existing files in the given path.
 
     :param bento_service (bentoml.service.BentoService): a Bento Service instance
-    :param path (str): Destination of where the bento service will be saved
+    :param path (str): Destination of where the bento service will be saved. The
```

Comment:
let's just add another method `save_to_tar`, and leave the upload to the user?

Otherwise you would also add support for s3/gcs in `load_from_dir` right?  It's quite confusing when `save_to_dir` is actually creating a tarball and upload the tarball to an object under the given s3 path, which is inconsistent with the saving to local directory behavior

Reply:
We actually already support loading from tarball in s3/gcs with `load_from_dir`. 

The inconsistent between saving to a local path or an remote path is not very desiring

Reply:
Add http location here as well then to make it consistent?